ops/flags/install-certs.yml

```diff
@@ -0,0 +1,37 @@
+- type: replace
+  path: /instance_groups/name=bosh/jobs/5:before
```

Comment:
does the order matter? As in does it has to be inserted before the 5th job? And if so wouldn't it be better to switch to `jobs/name=foo:before`?

Reply:
Initially I tried using `/instance_groups/name=bosh/jobs/-` and the UAA job failed - @ramonskie suggested to use `5:before` but I'm not sure what the justification was.

Reply:
no we can get rid of these changes since they merged it

Reply:
@ramonskie so are you saying that the whole file can just be

```
- type: replace
  path: /instance_groups/name=bosh/jobs/-
  value:
    name: ca_certs
    release: os-conf
    properties:
      certs: ((custom_ca_certs))
```

Reply:
@fenech yes that's all we need